### PR TITLE
doc: added readme and minor code fixes

### PR DESCRIPTION
**Pull request by Sahil25061999**
added readme file
readme: https://github.com/Sahil25061999/bijutsu-ecommerce/tree/readme

### CODE REVIEW

**Review by JuvitaSaini09 — APPROVED** [review]
read me images were not loading ,rest looks nice.I am approving

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+**To visit us go to : [Bijutsu Ecommerce](https://bijutsu.netlify.app/index.html 'Bijutsu Landing Page')**
+
+# Pages
+
+- [SignUp](https://bijutsu.netlify.app/pages/authentication/signup.html 'SignUp')
+- [Login](https://bijutsu.netlify.app/pages/authentication/login.html 'LogIn')
+- [Landing](https://bijutsu.netlify.app/index.html 'Bijutsu Landing Page')
+- [Product List](https://bijutsu.netlify.app/pages/product_listing/product_listing.html 'Product List')
+- [Wishlist](https://bijutsu.netlify.app/pages/wishlist/wishlist.html 'Wishlist')
+- [Cart](https://bijutsu.netlify.app/pages/cart/cart.html, 'Cart')
```
Comment (git-0r) [inline readme.md]: cart link not working, remove the ',' from the link

**Review by git-0r — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (Sahil25061999) [reply readme.md]: done

Comment (git-0r) [reply readme.md]: and footer has a lot of empty space because height is 40vh. Otherwise very good design.

**Review by AshutoshM29 — APPROVED** [review]
Looks good to me, can be merged!

**Comment by pradipgharge** [comment]
Readme looks good to me, can be merged !

**Review by pradipgharge — COMMENTED** [review]
good that you have added preview for the different features

**Pull request merged**